draft-ietf-trans-rfc6962-bis.md

```diff
@@ -690,6 +688,30 @@ Future versions of this protocol may reuse `VersionedTransType` values defined
 in this document as long as the corresponding data structures are not modified,
 and may add new `VersionedTransType` values for new or modified data structures.
 
+## Log Artifact Extensions
+
+~~~~~~~~~~~
+    enum {
+        reserved(65535)
+    } ExtensionType;
+
+    struct {
+        ExtensionType extension_type;
+        opaque extension_data<0..2^16-1>;
+    } Extension;
+~~~~~~~~~~~
+
+The `Extension` structure provides a generic extensibility for log entries,
```

Comment:
Nit: The term 'log entries' here is a bit vague: SCTs and STHs are not log entries. They are artifacts produced by the log. Maybe it'd be better to replace it with 'log artifacts' ?